src/main/java/pl/javagame/PasswordManager.java

```diff
@@ -1,35 +1,70 @@
 package pl.javagame;
 
 
-import java.util.ArrayList;
-import java.util.Random;
+import java.util.*;
 
 public class PasswordManager {
 
 
-    private ArrayList<String> passwords = new ArrayList<>();
+    private HashMap<String, Boolean> passwords = new HashMap<>();
+    // private ArrayList<String> copyPasswords = new ArrayList<>();
 
     public PasswordManager() {
-        passwords.add("Apetyt rośnie w miarę jedzenia");
-        passwords.add("Co dwie głowy to nie jedna");
-        passwords.add("Cwiczenie czyni mistrza");
-        passwords.add("Darowanemu koniowi w zęby się nie zagląda");
-        passwords.add("Diabeł tkwi w szczegółach");
-        passwords.add("Elektryka prąd nie tyka");
-
-        getRandomPassword();
+        passwords.put("Apetyt rośnie w miarę jedzenia", false);
+        passwords.put("Co dwie głowy to nie jedna", false);
+        passwords.put("Cwiczenie czyni mistrza", true);
+        passwords.put("Darowanemu koniowi w zęby się nie zagląda", false);
+        //    passwords.add("Diabeł tkwi w szczegółach");
+        //    passwords.add("Elektryka prąd nie tyka");
+
+        //    getRandomPassword();
     }
 
 
     public String getRandomPassword() {
-        int max = passwords.size(); // result=6
-        int i = (int) (Math.random() * max);
-        String randomPassword = passwords.get(i);
-        passwords.remove(i); // to avoid the same passwords
-        return randomPassword;
 
+        String Password = null;
+        String getKey;
```

Comment:
zmienne nazywamy od rzeczowników,  getKey i getValue to raczej dla metod nazwy. Może po prostu `currentKey` i `currentValue`?